condor-ap/condor_meter

```diff
@@ -893,6 +898,49 @@ def htcondor_configured():
     return False
 
 
+def get_collector_host():
+    """
+    Query for COLLECTOR_HOST
+    """
+    try:
+        path = GratiaCore.Config.getConfigAttribute("CondorLocation")
+        if g_probe_config is not None and 'htcondor-ce' in g_probe_config:
+          #condor_config_binary = 'condor_ce_config_val'
+          return None
+        else:
+          condor_config_binary = 'condor_config_val'
+        condor_config_path = os.path.join(path, "bin", condor_config_binary)
+        condor_config_val_args = ['COLLECTOR_HOST']
+
+        if (os.path.exists(condor_config_path) and
+            os.path.isfile(condor_config_path)):
+            args = [condor_config_path] + condor_config_val_args
+            cmd = subprocess.Popen(args, stdout=subprocess.PIPE)
+        else:
+            # args needs to be string when shell=True
+            args = condor_config_binary + ' ' + ' '.join(condor_config_val_args)
+            cmd = subprocess.Popen(args, stdout=subprocess.PIPE, shell=True)
+
+        cmd_stdout = cmd_stdout.strip()
+        if cmd.returncode != 0:
+            DebugPrint(-1, "WARNING: condor_config_val COLLECTOR_HOST "
+                           "returned a non-zero return code. Maybe the schedd "
```

Comment:
```suggestion
                           "returned a non-zero return code. Maybe the schedd "
```

I think `condor_config_val` queries the `condor_master` daemon by default

Reply:
ah, good catch

Reply:
Updated